content/03-type-system.md

```diff
@@ -475,41 +475,41 @@ In this example, we are actually importing a **module**, not just a specific typ
 
 [code asset](assets/Import2.hx)
 
-The type `Binop` is an [enum](types-enum-instance) declared in the module `haxe.macro.Expr`, and thus available after the import of said module. If we were to import only a specific type of that module, e.g. `import haxe.macro.Expr.ExprDef`, the program would fail to compile with `Class not found : Binop`.
+The type `Binop` is an [enum](types-enum-instance) declared in the module `haxe.macro.Expr`, and thus available after the import of said module. If we were to import only a specific type of that module, for example, `import haxe.macro.Expr.ExprDef`, the program would fail to compile with `Class not found : Binop`.
```

Comment:
This "e.g." thing is also something for our future style-guide.

Reply:
Do we collectively dislike e.g.?

Reply:
I have been removing it primarily because in most cases you're saving two words, either "for example" or "for instance," both of which are somewhat clearer.

If you'd prefer e.g. as a stylistic choice, I can revert similar changes back to it.

Reply:
I don't mind either way, but this strikes me as something where we should try to be consistent.